default.nix

```diff
@@ -1,14 +1,12 @@
 {pkgs ? import <nixpkgs> {}}:
-pkgs.rustPlatform.buildRustPackage rec {
+pkgs.rustPlatform.buildRustPackage {
   pname = "nyaa";
-  version = "0.9.0";
-
-  # src = fetchFromGitHub {
-  #   owner = "Beastwick18";
-  #   repo = "nyaa";
-  #   rev = "v${version}";
-  #   hash = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
-  # };
+  version =
+    (builtins.fromTOML (
+      builtins.readFile ./Cargo.toml
+    ))
+    .package
+    .version;
```

Comment:
You seemed to put it manually but it can be done like this too so it takes from `Cargo.toml` version variable